.eslintrc

```diff
@@ -15,6 +15,8 @@
     "no-unused-expressions": 0,
     "no-param-reassign": 0,
     "no-reserved-keys": 0,
-    "space-infix-ops": 0
+    "space-infix-ops": 0,
+    array-bracket-spacing: [2, "never"],
+    object-curly-spacing: [2, "never"]
```

Comment:
Nooo, I didn't mean to remove the spaces in curly braces `{}` as well, sorry 😅 Only in brackets `[]`. Normally the [`airbnb`-preset](https://github.com/airbnb/javascript/search?utf8=%E2%9C%93&q=array-bracket-spacing) should throw an error already.